src/AzureAppConfigurationImpl.ts

```diff
@@ -176,35 +176,70 @@ export class AzureAppConfigurationImpl implements AzureAppConfiguration {
         return {
             requestTracingEnabled: this.#requestTracingEnabled,
             initialLoadCompleted: this.#isInitialLoadCompleted,
-            appConfigOptions: this.#options
+            appConfigOptions: this.#options,
+            isFailoverRequest: this.#isFailoverRequest
         };
     }
 
-    async #loadSelectedKeyValues(): Promise<ConfigurationSetting[]> {
-        const loadedSettings: ConfigurationSetting[] = [];
+    async #executeWithFailoverPolicy(funcToExecute: (client: AppConfigurationClient) => Promise<any>): Promise<any> {
+        const clientWrappers = await this.#clientManager.getClients();
+        if (clientWrappers.length === 0) {
+            this.#clientManager.refreshClients();
+            throw new Error("No client is available to connect to the target app configuration store.");

```

Comment:
This behavior is different from .NET provider [ref](https://github.com/Azure/AppConfiguration-DotnetProvider/blob/main/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs#L232)

Reply:
updated